include/crypto/riscv_arch.h

```diff
@@ -26,7 +26,7 @@ uint32_t OPENSSL_riscvcap_P[ ((
 # endif
 
 # define RISCV_DEFINE_CAP(NAME, INDEX, BIT_INDEX)                   \
-    static inline int RISCV_HAS_##NAME(void)                        \
```

Comment:
I think HAS was more correct.  It reads better to me at least.

Reply:
I am used to "have" from too much of Autotools and its `config.h`. I saw the "RISCV" part just as a prefix/namespace.

But I don't have strong feelings, so I'll change back to HAS.